### PR TITLE
[6.5.x] DROOLS-2263 : Unexpected results in GDST when using enumerations with commas

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2263

This PR has a friend:
https://github.com/kiegroup/drools/pull/1716

### CODE REVIEW

**Comment by Rikkola** [comment]
And it needs acks.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java [hunk 1]
```diff
@@ -309,25 +310,30 @@ private String getTypeFromDataOracle( final String factType,
...
     private String[] getValueList( final ActionInsertFactCol52 col ) {
         if ( col.getValueList() != null && !"".equals( col.getValueList() ) ) {
-            return col.getValueList().split( "," );
+            return getSplit(col.getValueList());
         }
         return new String[ 0 ];
     }
 
+    private String[] getSplit(final String valueList) {
+        return ListSplitter.split(valueList);
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java]: Due to changes in `RuleModelDRLPersistenceImpl`, shouldn't we call `ListSplitter.split("\"",workingValue);` here?

Comment (Rikkola) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java]: I used the single ' quote for the outer quotes. This way " can be used inside. Here any " gets ignored and included.

Now I see a test in master says` "a, b b b  ,c"` should become `"a", " b b b  ", "c"`. Not sure if the ' is needed then.

Comment (Rikkola) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java]: Thinking thinking. Need to figure out how master and this branch are compatible.

Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/utils/GuidedDecisionTableUtils.java]: I mean, I understand that implementation in this PR, assumes *'* as spanner of options like *'John, Peter'*, and it works in UI as implemented, but I am not sure if it then affects somehow backend where *"* is used as a spanner of such options. I mean Imagine in UI *'"A", "B"'* will this backend store correctly?

**Comment by Rikkola** [comment]
Ok, the acks are in.

**Comment by manstis** [comment]
@Rikkola Need @jomarko to approve... is this for EoD today?

**Pull request merged**